doc/radosgw/iam.rst

```diff
@@ -61,6 +61,8 @@ Users
 +------------------------------+---------------------------------------------+
 | **ListAttachedUserPolicies** |                                             |
 +------------------------------+---------------------------------------------+
+| **GetAccountSummary**        |                                             |
++------------------------------+---------------------------------------------+
```

Comment:
instead of duplicating this for each table, maybe just create a separate `Accounts` table for it?